docs/06-contributors/999-rfcs/000-stories/story-13.md

```diff
@@ -113,14 +92,14 @@ resource TaskList {
     let output = MutArray<str>[];
     for id in task_ids {
       let j = this.get_task(id); 
-      let title = str.from_json(j.get("title"));
+      let title = j.get_str("title"); // notice this require us to create a small "get_str" function  implementation that should be removed by next sprint, maybe there is a different way around this. @hasanaburayyan and @staycoolcall911 ?
       if title.contains(term) { 
         print("found task ${id} with title \"${title}\" with term \"${term}\"");
         output.push(id);
       }
     }
     
-    print("found ${output.len} tasks which match term '${term}'");
+    print("found ${output.length} tasks which match term '${term}'");
```

Comment:
Why not `len`? Let's add support for getters

Reply:
I don't think we can accomplish this in this sprint

I'll add to the next one

Reply:
Note to self: update sprint 14 to have len instead of length